lib/gstreamer/va/encoder.py

```diff
@@ -72,7 +82,6 @@ def lowpower(self):
 
   gop     = property(lambda s: s.ifprop("gop", " key-int-max={gop}"))
   slices  = property(lambda s: s.ifprop("slices", " num-slices={slices}"))
-  bframes = property(lambda s: s.ifprop("bframes", " b-frames={bframes}"))
   minrate = property(lambda s: s.ifprop("minrate", " bitrate={minrate}"))
```

Comment:
@Bin-CI  move to  because av1 encode use different option
@property
  def bframes(self):
    def inner(bframes):
      if self.codec in ["av1-8"]:
        return " gf-group-size={bframes}"
      return f" b-frames={bframes}"
    return self.ifprop("bframes", inner)